cmd/auth/main.go

```diff
@@ -254,40 +263,83 @@ func newService(db *sqlx.DB, tracer opentracing.Tracer, secret string, logger lo
 	return svc
 }
 
-func startHTTPServer(tracer opentracing.Tracer, svc auth.Service, port string, certFile string, keyFile string, logger logger.Logger, errs chan error) {
+func startHTTPServer(ctx context.Context, tracer opentracing.Tracer, svc auth.Service, port string, certFile string, keyFile string, logger logger.Logger) error {
 	p := fmt.Sprintf(":%s", port)
-	if certFile != "" || keyFile != "" {
+	server := &http.Server{Addr: p, Handler: httpapi.MakeHandler(svc, tracer, logger)}
+	errCh := make(chan error)
+	protocol := "http"
```

Comment:
Please move string constants at the top, in the `const` section

Reply:
Hi @drasko ,



The variable protocol is reuse to find the service is started as HTTP or HTTPS 

By default variable `protocol` is set to `http` 
https://github.com/arvindh123/mainflux/blob/8accfe2c8dff57ca143c0924bfe0d872e60dd6e1/cmd/auth/main.go#L270



If the service start as HTTPS , https://github.com/arvindh123/mainflux/blob/8accfe2c8dff57ca143c0924bfe0d872e60dd6e1/cmd/auth/main.go#L277
then the  variable `protocol` is set to `https`


The detailed explanation is given in the [https://github.com/mainflux/mainflux/pull/1548#discussion_r835408692](https://github.com/mainflux/mainflux/pull/1548#discussion_r835408692
)

Reply:
Hi @drasko ,



The variable protocol is reuse to find the service is started as HTTP or HTTPS 

By default variable `protocol` is set to `http` 
https://github.com/arvindh123/mainflux/blob/8accfe2c8dff57ca143c0924bfe0d872e60dd6e1/cmd/auth/main.go#L270



If the service start as HTTPS , https://github.com/arvindh123/mainflux/blob/8accfe2c8dff57ca143c0924bfe0d872e60dd6e1/cmd/auth/main.go#L277
then the  variable `protocol` is set to `https`


The detailed explanation is given in the [https://github.com/mainflux/mainflux/pull/1548#discussion_r835408692](https://github.com/mainflux/mainflux/pull/1548#discussion_r835408692
)

Reply:
@manuio , I'm asking out of eager to know, Why we need to use in this way? .

Reply:
Because it's consistent with the rest of the code. You can also add a nice name to your caonstants to make them understandable.

Reply:
I think this is acceptable without constants and even more readable this way.